meta-sokol-flex-distro/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: MIT
 # ---------------------------------------------------------------------------------------------------------------------
 
-FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS:prepend:sokol-flex := "${THISDIR}/${PN}:"
 
-RDEPENDS:initramfs-module-lvm += "lvm2"
-RRECOMMENDS:${PN}-base += "initramfs-module-lvm"
+RDEPENDS:initramfs-module-lvm:sokol-flex += "lvm2"
+RRECOMMENDS:${PN}-base:sokol-flex += "initramfs-module-lvm"
```

Comment:
Same issue as finch, append is being converted to appending to an override which replaces the original rather than appending to it.